gnd/src/main/java/com/google/android/gnd/ui/editobservation/PhotoFieldViewModel.java

```diff
@@ -91,7 +91,11 @@ public void onShowPhotoSelectorDialog() {
     return showDialogClicks;
   }
 
-  public void setEditable(boolean editable) {
-    isEditable.postValue(editable);
+  public void setResponseCancelable(boolean enabled) {
```

Comment:
Since this is specific to photos, let's use a more obvious name here - perhaps `setRemovePhotoButtonVisible`?

Reply:
or "setClearButtonVisible()" for consistency.